Getting equal data (#30)
SpecialeBajs/Rapport: article/equal-data.tex

\subsection{Equal data} \label{equal-data}
When measuring the performance of the different methods it is essential that the datasets are the same.

> Without them being equal the results of the experiments wont be worth comparing.

Reviewer: ```suggestion
Without them being equal the results of the experiments will not be worth comparing.
```

Both PUP and LightGCN utilized the yelp dataset according to their papers.

> We did not get the dataset from the PUP implementation and were unable to find the original yelp-2018 dataset from the LightGCN implementation were the price and category was still attached.

Reviewer: ```suggestion
We did not get the dataset from the PUP implementation and were unable to find the original yelp-2018 dataset from the LightGCN implementation where the price and category were still attached.
```

We therefore decided to utilize the yelp-2020 dataset, in which users rank businesses between 1 to 5 stars.
Inspired by the PUP method we also decided to only use businesses with the restaurant category.

> What is left is set of businesses with one or more subcategories that relate to what type of restaurant they are.

Reviewer: ```suggestion
What is left is a set of businesses with one or more subcategories that relate to what type of restaurant they are.
```

> Like PUP we cut of any extra subcategories so that each business now only has one subcategory.

Reviewer: ```suggestion
Like PUP we cut off any extra subcategories so that each business now only has one subcategory.
```

The single subcategory that an item possesses is referred to as its category in the rest of this paper.
\\
PUP has a 60\% training-, 20\% validation- and 20\% test-data split while LightGCN has a 70\% training-, 10\% validation-, and 20\% test-data split.
It was decided to use the LightGCN split since this was the method we planned to extend.
\\

> After this we now have a dataset that can be used to compare the different methods with or without category and price.

Reviewer: ```suggestion
After this, we now have a dataset that can be used to compare the different methods with or without category and price.
```

\subsection{Equal data} \label{equal-data}
When measuring the performance of the different methods it is essential that the datasets are the same.
Without them being equal the results of the experiments wont be worth comparing.
Both PUP and LightGCN utilized the yelp dataset according to their papers.
We did not get the dataset from the PUP implementation and were unable to find the original yelp-2018 dataset from the LightGCN implementation were the price and category was still attached.

> We therefore decided to utilize the yelp-2020 dataset, in which users rank businesses between 1 to 5 starts.

Reviewer 2: ```suggestion
We therefore decided to utilize the yelp-2020 dataset, in which users rank businesses between 1 to 5 stars.
```

Inspired by the PUP method we also decided to only use businesses with the restaurant category.
What is left is set of businesses with one or more subcategories that relate to what type of restaurant they are.
Like PUP we cut of any extra subcategories so that each business now only has one subcategory.

> A future way to further extend PUP could be by having more than one subcategory for each item.

Reviewer 2: Måske ikke helt relevant til afsnittet men en god idé
